src/core/server/elasticsearch/client/mocks.ts

```diff
@@ -141,9 +141,10 @@ export type MockedTransportRequestPromise<T> = TransportRequestPromise<T> & {
 
 const createSuccessTransportRequestPromise = <T>(
   body: T,
-  { statusCode = 200 }: { statusCode?: number } = {}
+  { statusCode = 200 }: { statusCode?: number } = {},
+  headers?: Record<string, any>
```

Comment:
@joshdover thanks for pointing the type issue out!

Reply:
[done](https://github.com/elastic/kibana/pull/107301/files#diff-7a19c0f1ba03e2b736d8031dc6c5ea8b746cae37d2d81f357577ec34612dbc12R206)